proto/zigbeealliance/distributedcomplianceledger/model/tx.proto

```diff
@@ -69,11 +69,11 @@ message MsgUpdateModel {
   uint32 schemaVersion = 15 [(gogoproto.moretags) = "validate:\"gte=0,lte=65535\""];
   uint32 commissioningModeInitialStepsHint = 16;
   int32 enhancedSetupFlowOptions = 17 [(gogoproto.moretags) = "validate:\"omitempty,gte=0,lte=65535\""];
-  string enhancedSetupFlowTCUrl = 18 [(gogoproto.moretags) = "validate:\"required_if=EnhancedSetupFlowOptions 0,omitempty,url,startsnotwith=http:,max=256\""];
-  int32 enhancedSetupFlowTCRevision = 19 [(gogoproto.moretags) = "validate:\"required_if=EnhancedSetupFlowOptions 0,gte=0,lte=65535\""];
-  string enhancedSetupFlowTCDigest = 20 [(gogoproto.moretags) = "validate:\"required_if=EnhancedSetupFlowOptions 0,omitempty,max=128\""];
+  string enhancedSetupFlowTCUrl = 18 [(gogoproto.moretags) = "validate:\"required_if_bit_0_set=EnhancedSetupFlowOptions,omitempty,url,startswith=https:,max=256\""];
```

Comment:
I'm not sure required_if_bit_0_set is a right validation for the Update command.
All of these fields are optional.
We may have enhancedSetupFlowOptions not specified (assuming the old value is kept), and SOME of the  enhancedSetupFlow fields specified (to update the corresponding values). It seems the current validation will not allow such case.

Reply:
OK, actually we can keep the current logic, that in order to update one of ENHANCED fields, ALL fields must be present (or none of them).